src/sdkFactory/index.ts

```diff
@@ -65,7 +62,7 @@ export function sdkFactory(params: ISdkFactoryParams): SplitIO.ICsSDK | SplitIO.
 
   // trackers
   const observer = impressionsObserverFactory && impressionsObserverFactory();
-  const impressionsTracker = impressionsTrackerFactory(log, storage.impressions, metadata, impressionListener, integrationsManager, observer, storage.impressionCounts);
+  const impressionsTracker = impressionsTrackerFactory(log, storage.impressions, settings, impressionListener, integrationsManager, observer, storage.impressionCounts);
```

Comment:
Just a note: we should add a line max_char length to linter to make it easier to read.
```
  const impressionsTracker = impressionsTrackerFactory(
  log, 
  storage.impressions, 
  settings, 
  impressionListener, 
  integrationsManager, 
  observer, 
  storage.impressionCounts
);
```